examples/react-app/github/deploy.yml

```diff
@@ -0,0 +1,82 @@
+name: Architect Deployment
+
+on:
+  pull_request_target:
+    branches:
+      - master
+    types:
+      - closed
+
+env:
+  COMPONENT_TAG: latest
+  KUBECONFIG_NAME: eks-kubeconfig
+  ARTIFACT_NAME: architect-artifact
+  POSTGRES_HOST_FILENAME: postgres-host.txt
+  KUBERNETES_CLUSTER_NAME: kubernetes-cluster-name.txt
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  AWS_DEFAULT_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+
+jobs:
+  terraform_setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Set up terraform
+        uses: hashicorp/setup-terraform@v1.2.1
+        with:
+          terraform_version: 1.1.3
+          terraform_wrapper: false
+      - name: Initialize terraform
+        run: cd ./terraform && terraform init -backend-config="bucket=${{ secrets.REMOTE_STATE_BUCKET }}" -backend-config="region=${{ secrets.AWS_DEFAULT_REGION }}" -backend-config="key=${{ secrets.AWS_ACCESS_KEY_ID }}"
```

Comment:
Lets try to cleanup these commands. You already define the environment variables so this could be simplified to:

```
run: terraform init -backend-config="bucket=${{ env.REMOTE_STATE_BUCKET }}"
```

Bucket doesn't need to be a secret.